packages/vendor-core/package.json

```diff
@@ -28,9 +28,10 @@
     "@patternfly/react-catalog-view-extension": "^4.10.2",
     "@patternfly/react-core": "~4.97.2",
     "@patternfly/react-icons": "~4.9.2",
-    "@patternfly/react-styles": "~4.8.2",
+    "@patternfly/react-styles": "^4.10.2",
     "@patternfly/react-table": "~4.23.2",
-    "@patternfly/react-tokens": "~4.10.2",
+    "@patternfly/react-charts": "~6.14.17",
+    "@patternfly/react-tokens": "^4.11.3",
```

Comment:
Same for these, the matching version for react-charts is `@patternfly/react-charts@6.14.2`

Reply:
Updated..